app/controllers/reviews_controller.rb

```diff
@@ -21,7 +21,7 @@ def update
       flash[:notice] = "Review successfully update!"
       redirect_to "/shelters/#{@review.shelter_id}"
     else
-      flash[:alert] = "Please fill out all fields to update review."
+      flash[:error] = "Please fill out all fields to update review."
       render :edit
```

Comment:
We wanted to do the errors.full_messages.to_sentence but we stuck to the user story and created a controller flash message